### PR TITLE
Update advanced-build-settings-dialog-box-csharp.md

### PR DESCRIPTION
**Pull request by cartermp**
LangVersion selector is now disabled

### CODE REVIEW

**Comment by PRMerger17** [comment]
@cartermp : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by gewarren** [comment]
@cartermp Is this for 16.2? I don't see this in 16.1. We also share these docs with VS 2017 so we'll need to add monikered zones for both 2017 and 2019 if this is not changing in VS 2017.

**Comment by cartermp** [comment]
This will be for 16.3.

**Comment by cartermp** [comment]
How can we distinguish this doc between VS versions?

File: docs/ide/reference/advanced-build-settings-dialog-box-csharp.md [hunk 1]
```diff
@@ -23,19 +23,7 @@ The following options enable you to set general advanced settings.
...
-   Targets the current version.
-
-- **ISO-1** and **ISO-2**
-
-   Targets the ISO-1 and ISO-2 standard features, respectively.
-
-- **C# [version number]**
-
-   Targets a specific version of C#. For more information, see [/langversion (C# Compiler Options)](/dotnet/csharp/language-reference/compiler-options/langversion-compiler-option).
+Provides a link to [/langversion (C# Compiler Options)](/dotnet/csharp/language-reference/compiler-options/langversion-compiler-option) so you can learn why your default language version was chosen for you based on your target framework.
```
Comment (gewarren) [inline docs/ide/reference/advanced-build-settings-dialog-box-csharp.md]:
```suggestion

moniker range=">=vs-2019"

Provides a link to [/langversion (C# Compiler Options)](/dotnet/csharp/language-reference/compiler-options/langversion-compiler-option) so you can learn why your default language version was chosen for you based on your target framework.

::: moniker-end

::: moniker range="vs-2017"

(The VS 2017 version info would go here.)

::: moniker-end
```

**Comment by gewarren** [comment]
> How can we distinguish this doc between VS versions?

I left a suggestion showing how to do this. However, the best thing would be to move this change over to the private repo (visualstudio-docs-pr instead of visualstudio-docs) and create a PR against the release-16.3 branch.

**Comment by cartermp** [comment]
Okay. I'll take these changes over there.

**Comment by PRMerger7** [comment]
@cartermp : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Pull request closed without merging**